CONTENT/about.markdown

```diff
@@ -1,6 +1,6 @@
 [//]: # (@TODO - make this file nice)
 
-# Manifesto
+# About Wizard Amigos Institute
 (Philosophy/Values/Goals)
 => Link to Github
```

Comment:
Every Section - as you might have realized in other markdown files starts with:
# A Title
## A Subtitle
...then comes the content...